CONTRIBUTORS.md

```
@@ -0,0 +1,274 @@
+ - [allesmi](https://github.com/allesmi)
+
+# Emby Contributors
+
+ - [LukePulverenti](https://github.com/LukePulverenti)
+ - [ebr11](https://github.com/ebr11)
+ - [lalmanzar](https://github.com/lalmanzar)
+ - [schneifu](https://github.com/schneifu)
+ - [Mark2xv](https://github.com/Mark2xv)
+ - [ScottRapsey](https://github.com/ScottRapsey)
+ - [skynet600](https://github.com/skynet600)
+ - [Cheesegeezer](https://githum.com/Cheesegeezer)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)